validation/static/helpers.js

```diff
@@ -0,0 +1,22 @@
+"use strict";
+
+// Helper functions that are needed across multiple JS files
+
+
+function getElementByPhraseId(className, phraseId) {
```

Comment:
You should be able to do a lookup using the DOM API instead of having to do a linear scan:

    document.querySelector(`.${className}[data-phrase-id=${phraseId}]`)

That said, I know there were issues with quoting related to the format of the ID that may make that trickier. That’s one reason that automatic numeric ID columns are so handy.